src/Tribe/Featured_Events/Permalinks_Helper.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * @internal
+ */
+class Tribe__Events__Featured_Events__Permalinks_Helper {
+	public function hook_up() {
```

Comment:
I don't have a super strong opinion and would be delighted to standardize. Of the suggestions you put forward I dislike `hook()` singular most of all. I also wonder if we should consider `hooker()`.